frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -168,6 +215,9 @@ export const VMStatus: React.FC<VMStatusProps> = ({ vm, vmi, vmStatusBundle }) =
           <ImporterPods key="importerPods" statuses={importerPodsStatuses} />
         </VMStatusPopoverContent>
       )}
+      {arePendingChanges && (
```

Comment:
same here let's init all these things in running clause

Reply:
what to you mean? can you please elaborate ?

Reply:
that this render should be valid only when vm is running, or otherwise we should combine the pending changes with other statuses as well